tests/10apidoc/01register.pl

```diff
@@ -61,6 +61,32 @@
       });
    };
 
+test "POST /createUser can get or create a user",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   critical => 1,
+
+   do => sub {
+      my ( $http ) = @_;
+
+      $http->do_request_json(
+         method => "POST",
+         uri    => "/api/v1/createUser",
```

Comment:
This should probably be using the `/unstable/createUser` url, or whatever it is.